resources/upgrading-to-aep/current-sdk-versions.md

```diff
@@ -171,6 +171,39 @@ For React Native, we recommend that you first install [Node.js](https://nodejs.o
 Beta versions of React Native Experience Platform Edge Network, Edge Identity, Edge Consent, Messaging and Assurance can be found in the repository link [here](https://github.com/adobe/aepsdk-react-native). These new beta extensions require the latest AEP Core to opperate. See [`README`](https://github.com/adobe/aepsdk-react-native/blob/main/README.md) for further information on installation and configuration.
 {% endhint %}
 
+## Flutter (AEP 1.x) <a id="flutter-aep"></a>
+
+Adobe Experience Platform Mobile SDK plugin for Flutter supports Flutter **versions 2.0.0 or later**.
+
+{% hint style="danger" %}
+Adobe Experience Platform Mobile SDK plugins for Flutter are compatible with [Android](current-sdk-versions.md#android) and [iOS — Swift ](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions#ios-swift) libraries, at this time.
+{% endhint %}
+
+| Extension | pub.dev | Github | Sample App |
```

Comment:
similar with the react native aep section, can we drop the github and sample app columns and add a mention add a mention under the header with the link to the flutter repo?